src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -34,10 +34,77 @@ public static IServiceCollection AddMvc([NotNull] this IServiceCollection servic
         {
             ConfigureDefaultServices(services);
 
+            AddMvcServices(services);
+
+            return services;
+        }
+
+        /// <summary>
+        /// Configures a set of <see cref="MvcOptions"/> for the application.
+        /// </summary>
+        /// <param name="services">The services available in the application.</param>
+        /// <param name="setupAction">The <see cref="MvcOptions"/> which need to be configured.</param>
+        public static void ConfigureMvc(
```

Comment:
one is related to changes from `TryAdd` to use `Add` and the other is pulling out services specifically added by MVC to enable unit testing.